rqt_robot_dashboard/src/rqt_robot_dashboard/monitor_dash_widget.py

```diff
@@ -48,6 +48,8 @@ class MonitorDashWidget(IconToolButton):
     :param context: The plugin context to create the monitor in.
     :type context: qt_gui.plugin_context.PluginContext
     """
+    msg_trigger = pyqtSignal()
```

Comment:
Since the signal shouldn't be used externally it should also be prefixed with an underscore to make it "private".